packages/validator/src/validator.ts

```diff
@@ -159,6 +162,24 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        if (!indicesService.validatorPubKeyExists(pubkeyHex)) {
+          this.logger.info("Validator exists in beacon chain", {
```

Comment:
won't this still print out `Validator exists in beacon chain` for each validator?

definitely looking forward to having the fee recipient printed out 🙌

Reply:
It shouldn't, given that a validator index would have existed in the `pubkey2index` Mapping had it been discovered prior to this. Therefore it would only be for newly discovered validators by the indices service.

Reply:
that's if the service is already running and we import and new validator I would assume but what happens if the service starts, isn't `pubkey2index` just stored in-memory?

Reply:
Yes in that case it would print it for every validator, but I would suggest that's technically accurate since at the start of the service the validator client would be discovering all indices since they would all be new to the client.

Reply:
Sorry I think I had misinterpreted what was written there, following the [lighthouse ux](https://github.com/ChainSafe/lodestar/issues/4785#issuecomment-1324702738) I think we should print this out for each validator, I do agree it can be a bit noisy upon initialization but I think it's useful information.

Reply:
It definitely is needed information if we discover a new validator while the VC is running I am not so sure at startup, I could imagine a log where if validators are > 2 we print out `Validators exists in beacon chain`, followed by a list of validators (index, pubkey, fee recipient). 

We can definitely also print this out for each and gather more feedback from users, I don't think there is a right or wrong way to do it.

Reply:
Good point, I think since it was originally agreed upon to be this way let's move forward with this and then see what the user feedback is